scholia/app/templates/base.html

```diff
@@ -60,7 +60,19 @@
     $.getJSON(url, function (data) {
 	 var item = data.entities["{{ q }}"];
 	 if ('en' in item.labels) {
```

Comment:
perhaps this should be making use of ```entityToLabel``` from scholia.js.

Reply:
Isn't that outside the PR?